taxonomies/ingredients_processing.txt

```diff
@@ -1645,6 +1684,120 @@ wikidata:en: Q7601513
 #fr:germe, gluten, amidon
 #hr:klica, gluten, škrob
 
+en: condensed
+af: Kondens
+ar: المبخر
+az: Qatılaşdırılmış
+be: згушчанае
+bg: кондензирано
+ca: condensada
+cs: kondenzované
+da: kondenseret
+de: Kondens, kondensierte, dosen, büchsen
+el: συμπυκνωμένο
+eo: kondensita
+es: condensada
+et: kondens
+eu: lurrundua, kondentsatu
+fa: تغلیظ‌شده
+fi: kondensoitu
+fr: condensé, condensée, condensés, condensées
+gl: condensado
+he: מרוכז
+hr: kondenzirano, ugušćeno
+hu: sűrített, sűrtett, sűrítmény
+hy: Խտացրած
+id: kental
+it: condensato
+kn: ಕಂಡೆನ್ಸ್‌ಡ್
+ko: 연
+lb: kondenséiert
+lt: sutirštintas, kondensuotas
+lv: iebiezināts, iebiezinātais
+nb: kondensert
+nl: gecondenseerde
+pl: skondensowane
+pt: condensada, condensado
+ro: condensat
+ru: сгущенное, сгущённое
+sl: kondenzirano
+sv: kondenserad
+th: ข้น
+tl: na kondensada
+tr: Kondense
+tt: Куертылган
+uk: згущене
+vi: đặc
+zh: 煉
+
+# description:en:LOW-FAT MILK is a 1% fat milk, sold in the United States
+# description:en:Skimmed milk (British English) is made when all the cream is removed from whole milk. It tends to contain around 0.1% fat. Láit écrémé has a maximum of 0.5% fat.
+# description:en:Skim milk (American English), is made when all the cream is removed from whole milk. It tends to contain around 0% fat.
+# Nonfat, skim milk and skimmed milk are often used as synonyms and the
+# regulatory differences are small, with the average being between 0.0% and
+# 0.2% fat. We are thus using only one taxonomy entry for all milks with
+# a maximum of 0.5% fat
+
+en: skimmed, skim, non-fat, nonfat, fat-free, low-fat, lowfat, defatted, 1% fat, with 0.5% milk fat
+ar: منزوع الدسم
+bg: обезмаслено, обезмаслно, обезмаслен, обезмаслена, обезмаслени
+ca: desnatada, descremada, desgreixat, desgreixada, desgreixats, desgreixades, sense greix, baixa en greixos
+cs: odstředěné, odstředěné
+da: skummet
```

Comment:
“skummet” _can_ refer to skimming (which implies removing the froth), but generally refers to the result of any process that creates froth (e.g., whipping cream or frothing coffee).

```suggestion
```